samples/basic/rgb_led/README.rst

```diff
@@ -8,13 +8,11 @@ Overview
 
 This is a sample app which drives a RGB LED using PWM.
 
-There are three LEDs of single color in a RGB LED. The
-three LEDs will be driven by a pwm port each. The pulse
-width for each pwm port will change from zero to period.
-So, the brightness for each LED will change from dark to
-max brightness in 11 steps. The three "for" loops (one
-for each LED) will generate 1331 combinations and so,
-1331 different colors. The whole process will repeat for
+There are three LEDs of single color in a RGB LED. The three LEDs will be driven
```

Comment:
I know you just reformatted the text here, but the original text doesn't read well.  How about:
```
There are three single-color component LEDs in an RGB LED. Each component
LED is driven by a PWM port where the pulse width is changed from zero to a
fusion flicker threshold (the minimum flicker rate where the LED is perceived
as being steady), causing each component LED to step from dark to max
brightness. Three **for** loops (one for each component LED) generate a
gradual range of color changes from the RGB LED, and the sample repeats forever.
```